scripts/reviews_scraper.js

```diff
@@ -1,45 +1,50 @@
-const fs = require('fs');
-const axios = require('axios');
-const {promisify} = require('util');
-const {apiKey} = require('./config');
-const {existsFile, sleep} = require('./util');
+/* eslint-disable id-length,no-await-in-loop */
+const fs = require("fs");
+const axios = require("axios");
+const {promisify} = require("util");
+const {apiKey} = require("./config");
+const {existsFile, sleep} = require("./util");
 
 const readFilePromise = promisify(fs.readFile);
 const writeFilePromise = promisify(fs.writeFile);
 const deleteFilePromise = promisify(fs.unlink);
 
-module.exports.scrape = async (recipeID, pageSize = 10) => {
+module.exports.scrape = async(recipeID, pageSize = 10) => {
     let pageNumber = 0;
-    let reviews;
+    let reviews = null;
     const idRatings = {
-        reviews: [],
-        ratings: {}
+        ratings: {},
+        reviews: []
     };
-    for (let i = 5; i > 0; i--) {
+
+    for (let i = 5; i > 0; i -= 1) {
         idRatings.ratings[`${i}-star`] = 0;
     }
 
     do {
         try {
             const fileName = getPath(recipeID, pageNumber);
+
+            // eslint-disable-next-line prefer-destructuring
             reviews = (await getReviews(recipeID, pageNumber, pageSize)).data.reviews;
-            for (let i = 0; i < reviews.length; i++) {
+            for (let i = 0; i < reviews.length; i += 1) {
```

Comment:
why no ++?

Reply:
Does the linting demand this?

Reply:
yee

Reply:
although we could turn it off so it doesn't require it